distributions/standalone/src/main/resources/conf/scheduler-pools.conf

```diff
@@ -0,0 +1,27 @@
+# Configures the thread pools of the Mule Runtime
+# Certain attributes support expressions. Those are arithmetic expressions (+, -, / or * operators supported only) that are
+# evaluated when the runtime starts.
+# Those expressions may be applied to numbers (integers or decimals) or any of the following variables:
+# * C: the number of processors available to the Java virtual machine.
+
+# The maximum time to wait until all tasks in all the runtime thread pools have completed execution when stopping the
+# scheduler service.
+SchedulerService.gracefulShutdownTimeoutSeconds=15
+
+# The number of threads to keep in the cpu_lite pool, even if they are idle.
+# Supports Expressions
+SchedulerService.cpuLight.threadPoolSize=2*C
```

Comment:
any reason not to use `2*cores`.  It more readable and more obviously apparent what it is.